src/backends/torch/torchgraphbackend.cc

```diff
@@ -163,7 +163,12 @@ namespace dd
             std::get<1>(std::get<1>(full_output))); // last memory / c  value
         if (_lstm_continuation)
           {
-            _rnn_memories[opname_v] = std::get<1>(full_output);
+            if (_rnn_has_memories[opname_v])
+              _rnn_memories.erase(opname_v);
+            _rnn_memories.insert(
```

Comment:
Just to make sure, this is not related to this PR, right ?

Reply:
no, removed